experimental/dds/attributable-map/src/test/mocha/map.spec.ts

```diff
@@ -967,6 +971,36 @@ describe("Map", () => {
 					"The loaded map should have valid op-based attribution for the second entry",
 				);
 			});
+
+			it("can update attribution properly while applying stashed ops", async () => {
```

Comment:
something to think about--I probably would have written this as 3 separate tests. This gives clearer failures for issues in different parts of the code.

mostly noting this for future consideration; I don't care much that you fix it as part of this PR.